File: (8 kyu) Even or Odd.js
/** Even or Odd (8 kyu)
 * https://www.codewars.com/kata/even-or-odd
 * Create a function that takes an integer as an argument and returns
 * "Even" for even numbers or "Odd" for odd numbers.
 */
// #1
//  const even_or_odd = (number) => (number % 2 ? 'Odd' : 'Even');
// #2
const even_or_odd = (n) => ['Even', 'Odd'][n & 1];

File: 8kyu/(8 kyu) Even or Odd/(8 kyu) Even or Odd.py
def even_or_odd(number):
    # #1
    # if number % 2 == 0:
    #     return "Even"
    # else:
    #     return "Odd"
    # #2
    # return "Odd" if number % 2 else "Even"
    # #3
    return ("Odd", "Even")[number % 2]
